.github/workflows/testsuite.yml

```diff
@@ -16,81 +16,94 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
+
     - name: Cache dependencies
-      uses: actions/cache@v2
+      uses: actions/cache@v4
       with:
         path: ~/.m2/repository
         key: ${{ runner.os }}-clj-${{ hashFiles('**/project.clj') }}
         restore-keys: |
           ${{ runner.os }}-clj-
+
     - name: Setup Java ${{ matrix.jdk }}
-      uses: actions/setup-java@v1.4.3
+      uses: actions/setup-java@v4
       with:
+        distribution: temurin
         java-version: ${{ matrix.jdk }}
+
     - name: Setup Clojure
-      uses: DeLaGuardo/setup-clojure@3.1
+      uses: DeLaGuardo/setup-clojure@13.1
       with:
         lein: 2.9.5
-    - name: Install dependencies
-      run: |
-        npm ci
+
```

Comment:
I think I had the `npm ci` here to install the locked version of `@seriousme/openapi-schema-validator`. Otherwise I think the `npx` command in `reitit.openapi-test` will grab the latest version and the tests won't be repeatable.